### PR TITLE
RavenDB-5798

### PR DESCRIPTION
**Pull request by IdanHaim**
- update the disable and enable index admin endpoint to be able to disable/enable all the indexes in database/indexes/indexes.html
- Fix the databaseConfigure to be able to parse bool fields
- separate between pause indexes and disable them
- add disable indexing button in studio

**  look here http://issues.hibernatingrhinos.com/issue/RavenDB-6088

### CODE REVIEW

**Comment by ppekrol** [comment]
I think we need to:
a) unify configuration handling: http://issues.hibernatingrhinos.com/issue/RavenDB-4489 now we have a same code for changing the configuration in multiple places
b) check if we can disable indexing without reloading the database
c) check if we can disable indexing without loading offline database (and then reloading it)

**Comment by ml054** [comment]
I skimmed though PR and have few comments on studio part. I describe them tomorrow. Please don't merge yet.

**Comment by IdanHaim** [comment]
@ml054 we will have to speak tomorrow , have some ideas for the UI
@ppekrol when did you find this issue?
It will be nice not to reload it again but I couldn't find a good way to do it. (look at the disable database)

File: src/Raven.Server/Documents/Handlers/Admin/AdminIndexHandler.cs [hunk 1]
```diff
@@ -137,10 +138,81 @@ public Task Start()
             return Task.CompletedTask;
         }
 
+        private void ToogleIndexesStatus(bool enableRequest)
```
Comment (ayende) [inline src/Raven.Server/Documents/Handlers/Admin/AdminIndexHandler.cs:141]: We are going to do that only in AdminDatabases

**Pull request closed without merging**